packages/config-dug/test/config-dug.test.ts

```diff
@@ -0,0 +1,53 @@
+import { z } from 'zod';
+
+import { ConfigDug } from '../src/config-dug';
+
+describe('ConfigDug', () => {
+  const testConfigSchema = {
+    testNum: z.number().default(0),
+  };
+
+  describe('getConfig', () => {
+    const configDug = new ConfigDug(testConfigSchema);
+
+    it('should throw an error when attempting to call getConfig without first loading', async () => {
+      await expect(async () => configDug.getConfig()).rejects.toThrowError(
```

Comment:
```suggestion
      await expect(configDug.getConfig()).rejects.toThrowError(
```

Might be a dumb question, but do we need this extra promise?

Reply:
Yeah I thought we didn't need this either but it doesn't properly catch/expect the error. The error still throws but the test fails because of it 🤔